reconcile/openshift_base.py

```diff
@@ -128,6 +128,51 @@ def clusters(
     ) -> list[str]: ...
 
 
+def validate_managed_resource_types(
+    oc: OCCli,
```

Comment:
only `oc.api_resources` are used, should we just pass it instead of the whole `oc` object?

Reply:
See comment below